### ISSUE TITLE
FaceIt AC Problems

### ISSUE THREAD

**Issue by RodgerTheDodger123** [Jire/Charlatano#78]
The cheat no longer works when the FaceIt anti-cheat is running on my computer. In order to play FaceIt the anti-cheat must be running or you'll be kicked from the server. The cheat simply does not work with it open although it does not get you banned while its opened.

**Comment by kokolino1** [comment]
yeah it doesn't work when faceit ac is running even with the latest version 0.9.8
But jire said : Safe aim is safe to use on FaceIT/ESEA,  ESEA has AC right ?
If the cheat can bypass esea's ac, then why it couldn't do the same with faceit ac ? even when using the safe aim.

Regards.
